reflex/components/radix/themes/components/tabs.py

```diff
@@ -59,8 +62,18 @@ class TabsTrigger(RadixThemesComponent):
     # Whether the tab is disabled
     disabled: Var[bool]
 
+    # The color of the line under the tab when active.
+    color_scheme: Var[LiteralAccentColor]
+
     _valid_parents: List[str] = ["TabsList"]
 
+    def _apply_theme(self, theme: Component):
```

Comment:
Should this go to `add_style` afterwards?

Reply:
actually, i think i'll do it in `create`